src/styles/blocks/header.scss

```diff
@@ -0,0 +1,26 @@
+
+.header {
+  position: relative;
+  width: 100%;
+  max-width: 1200px;
```

Comment:
Make header fullwidth and make flexible with justify-content: space-between(from previous task)